<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>615</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>945</width>
    <height>614</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>945</width>
    <height>615</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Time-Domain Plot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabTimDomain">
       <attribute name="title">
        <string>Time-Domain Plot</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_TimeDomain">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>111</width>
          <height>161</height>
         </rect>
        </property>
        <property name="title">
         <string>Horizontal Setting</string>
        </property>
        <widget class="QComboBox" name="comboBox_TriggerDomain">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>91</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Auto</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>External</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_Trigger_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Trigger:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_SampleNumber">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Sample Rate:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_SampleRate">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>51</width>
           <height>20</height>
          </rect>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>500</string>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>90</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>MS/s</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_RecordLength">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>51</width>
           <height>20</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Please input 2* like 4/8/16... from 2 to 1024*1024</string>
         </property>
         <property name="text">
          <string>4</string>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_SampleNumber_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Record Length:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>130</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>K</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="widget_Signal_TimeDomain" native="true">
        <property name="geometry">
         <rect>
          <x>119</x>
          <y>10</y>
          <width>791</width>
          <height>561</height>
         </rect>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Stop_TimeDomain">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>510</y>
          <width>82</width>
          <height>23</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>82</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Start_TimeDomain">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>480</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_TimeDomain_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>180</y>
          <width>111</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>Vertical Setting</string>
        </property>
        <widget class="QLabel" name="label_SampleNumber_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Voltage Scale:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_VolScale">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>41</width>
           <height>20</height>
          </rect>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>200</string>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>50</y>
           <width>41</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>mV/Div</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_Offset">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>41</width>
           <height>20</height>
          </rect>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_SampleNumber_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Offset:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>110</y>
           <width>41</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>mV</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_TimeDomain_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>330</y>
          <width>111</width>
          <height>91</height>
         </rect>
        </property>
        <property name="title">
         <string>Frame Mode:</string>
        </property>
        <widget class="QLabel" name="label_SampleNumber_4">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string> Number:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>41</width>
           <height>20</height>
          </rect>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>10</string>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>70</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Enable</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton_Save_TimeDomain">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>540</y>
          <width>82</width>
          <height>23</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>82</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_CHA">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>424</y>
          <width>82</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Channel A</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_CHB">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>450</y>
          <width>82</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Channel B</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="action3G">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3G</string>
   </property>
  </action>
  <action name="action100M">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>100M</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
